### ISSUE TITLE
Error! OH NO!!! Cheater detect!!! Turn off the plug-in or bypass software (for example, Universal Bypass, EasyList ..etc) and try again.

### ISSUE THREAD

**Issue by wisnumurti1993** [Sainan/Universal-Bypass#1957]
Some website can detect universal bypass, like the one below. Hopefully it can be fixed
![Screenshot_2021-05-05-07-10-15-003_com android chrome](https://user-images.githubusercontent.com/58716971/118351464-659d8800-b586-11eb-8cc2-66397755c30f.jpg)

https://fstore.biz/file/9372294802ce9fEb9C2DC97AAa827992

**Comment by Toshiba1127** [comment]
Duplicate of https://github.com/Sainan/Universal-Bypass/issues/1935

It can't be fixed because server-sided. However don't forget to use uBlock origin to block ads

**Comment by Toshiba1127** [comment]
@wisnumurti1993 also what browser are you using? i thought it's not possible to install plugins on Android?

**Comment by wisnumurti1993** [comment]
It's on Google Chrome on PC. The screenshot on phone, just something I found on the web for an example

Sent from Yahoo Mail on Android 
 
  On Sat, 15 May 2021 at 19:19, ***@***.***> wrote:   


@wisnumurti1993 also what browser are you using? i thought it's not possible to install plugins on Android?

**Comment by Toshiba1127** [comment]
Oh ok

**Comment by elemonader** [comment]
> @wisnumurti1993 also what browser are you using? i thought it's not possible to install plugins on Android?

o_O

"Impossible is nothing!"

![Screenshot_20210518-055708 cleaned](https://user-images.githubusercontent.com/71942553/118640832-f4014c00-b7c8-11eb-861e-91c118d5247c.jpg)

**Comment by Toshiba1127** [comment]
@elemonader yeah haha. Btw, are you using Firefox? how can you install unsupported add-ons other than ones listed on latest version? I'm using old Firefox 68 which is the last to support, it's very outdated and runs quite slower

PS: Sorry for off-topics, i'm just curious

**Comment by elemonader** [comment]
@Toshiba1127 Ah yes, you'd definitely want to move on from 68 if not just for the security updates. The standard method:

-DL Firefox Nightly
-5x tap on the logo in "about" sub-menu to activate "developer options"
-this will display a new menu option: "Custom Add-ons Collection"

From that point, you can either sign in at Mozilla.com to create your own "collection" of apps to use or, bc collections are shared publicly, you can simply input the info for an already curated collection skip that step. My app collection can be used by entering:

ID: 16844964
NAME: DegenerateBeast

I don't use most of the add-ons in my collection as many were included for testing. If you'd like for me to add any others just lmk, I'd be happy to do so. It should be noted, some do not function at all. Also, I mentioned that this is the "standard" method...

But I actually use a fork from @SkewedZeppelin and Divested-Mobile called "Mull-Fenix" instead. It's available here on GitHub or in their repo at F-Droid and receives timely updates. Mull has been deblobbed of proprietary bits and enables some privacy fearures* upstreamed by the Tor Uplift project via Arkenfox user.js (formerly known as ghacks) about:config prefs.

*Caveat emptor:
Many (most?) of these about:config switches have not been fully tested within the mobile platform. As a matter of fact the Arkenfox maintainer, @Thorin-Oakenpants, explicitly warns against using them as such. Citing the vast differences between the desktop and mobile client frameworks. As an experimental mobile-browser, I'm satisfied with the functionality but ymmv.

Anyway, here is a regularly updated list of popular F-Droid repos:
https://forum.f-droid.org/t/known-repositories/721

DivestOS is in there for Mull and I highly recommend the @IzzySoft repo, IzzyOnDroid.

**Comment by Toshiba1127** [comment]
@elemonader woah didn't know that. I just created my collections, but sadly some add-ons can't be installed like video download, but can be installed on Firefox 68. So i'll keep both 68 and nightly version

**Comment by elemonader** [comment]
For video downloads within Android, I'm quite fond of DVD maintained by @yausername ... It's based on youtube-DL and I've yet to find a site it won't pull video from.

**Comment by Toshiba1127** [comment]
Alright thanks
